lib/bgsite_official_web/templates/request/index.html.eex

```diff
@@ -18,6 +18,7 @@
       <!--<td><%= request.name %></td>
       <td><%= request.email %></td>-->
       <td><%= request.url %></td>
+      <td><%= @current_user.email %></td>
```

Comment:
This way the line shows us current user's email - not the sender's. E.g. if you sign in with user "Gosho", all "@current_user.email"s will be gosho@abv.bg, but if we sign in with "Pesho" the email won't remain "gosho@abv.bg". And so called, admins can't open the page, coz the output would be "function nil.email is undefined"